projects/samples/robotbenchmark/inverted_pendulum/worlds/inverted_pendulum.wbt

```diff
@@ -18,8 +18,38 @@ Viewpoint {
   follow "e-puck"
 }
 TexturedBackground {
-}
-TexturedBackgroundLight {
+  texture "empty_office"
```

Comment:
We could add a background of an indoor environment such as this one:
https://hdrihaven.com/hdri/?c=indoor&h=entrance_hall